src/StockMarket/ui/InfoAndPurchases.tsx

```diff
@@ -125,23 +138,35 @@ function PurchaseTixApiAccessButton(props: IProps): React.ReactElement {
         TIX API Access <CheckIcon />
       </Typography>
     );
-  } else {
-    const cost = StockMarketConstants.TixApiCost;
-    return (
-      <Button disabled={!Player.canAfford(cost) || !Player.hasWseAccount} onClick={purchaseTixApiAccess}>
-        Buy Trade Information eXchange (TIX) API Access -&nbsp;
-        <Money money={cost} forPurchase={true} />
-      </Button>
-    );
   }
+  const cost = StockMarketConstants.TixApiCost;
+  let tooltipTitle = "Let you trade stock through Netscript";
+  if (!Player.hasWseAccount) {
```

Comment:
If you check the netscript functions, purchasing tix access doesn't require a wse account. Purchasing 4s data tix access doesn't require 4s data. Maybe those things should be changed, but I don't think they should be changing in this PR.